src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Dialog.kt

```diff
@@ -20,26 +20,114 @@ package io.github.yuutoproject.yuutobot.commands
 
 import io.github.yuutoproject.yuutobot.commands.base.AbstractCommand
 import io.github.yuutoproject.yuutobot.commands.base.CommandCategory
+import io.github.yuutoproject.yuutobot.utils.EMOJI_REGEX
+import io.github.yuutoproject.yuutobot.utils.EMOTE_MENTIONS_REGEX
+import io.github.yuutoproject.yuutobot.utils.NONASCII_REGEX
 import java.io.IOException
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
 import net.dv8tion.jda.internal.utils.IOUtil
 import okhttp3.*
 import okhttp3.Callback as OkHttp3Callback
 import okhttp3.MediaType.Companion.toMediaType
 import okhttp3.RequestBody.Companion.toRequestBody
+import org.slf4j.LoggerFactory
 
-// TODO: get arch to implement the actual command
 class Dialog : AbstractCommand("dialog", CommandCategory.INFO, "does something", "[bg] <char> <text>") {
+    private val logger = LoggerFactory.getLogger(this.javaClass)
+
+    private val backgrounds: List<String> = listOf(
+        "bath",
+        "beach",
+        "cabin",
+        "camp",
+        "cave",
+        "forest",
+        "messhall"
+    )
+
+    private val characters: List<String> = listOf(
+        "aiden",
+        "avan",
+        "chiaki",
+        "connor",
+        "eduard",
+        "felix",
+        "goro",
+        "hiro",
+        "hunter",
+        "jirou",
+        "keitaro",
+        "kieran",
+        "knox",
+        "lee",
+        "naoto",
+        "natsumi",
+        "seto",
+        "taiga",
+        "yoichi",
+        "yoshi",
+        "yuri",
+        "yuuto"
+    )
+
+    private val backgroundsString: String = backgrounds.joinToString("`, `")
+    private val charactersString: String = characters.joinToString("`, `")
+
+    // Using ExperimentalStdLib for .removeFirst() in Mutable Lists
+    @ExperimentalStdlibApi
     override fun run(args: MutableList<String>, event: GuildMessageReceivedEvent) {
         val moveThisToAStorageLocation = OkHttpClient()
```

Comment:
This variable is named like that for a reason, don't create a new HTTP client on every command call